vagrant/Vagrantfile

```diff
@@ -93,6 +103,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     v.memory = 1024
     v.cpus = 1
     override.vm.box = box_name
+    if override_box_url
+      override.vm.box_url = override_box_url
+    end
```

Comment:
I see a virtualbox image in the root so probably something strange happening locally.